examples/glamorous-jest-react/index.test.js

```diff
@@ -0,0 +1,22 @@
+import {matcher, serializer} from 'jest-glamor-react'
+import React from 'react'
+import * as enzyme from 'enzyme'
+import toJson from 'enzyme-to-json'
+import {Wrapper, Title} from './index'
+
+expect.addSnapshotSerializer(serializer)
```

Comment:
Could you add a comment above this line that says:

```
// add the serializer and `toMatchSnapshotWithGlamor` matcher
// you can do this in a setup file instead so you don't have to do it
// for all of your tests
// https://facebook.github.io/jest/docs/configuration.html#setupfiles-array
```

Either that or maybe we should just use a setup file. Whichever you prefer :)